package.json

```
@@ -88,7 +91,7 @@
     "build:remix": "remix build",
     "dev": "run-p dev:*",
     "dev:css": "yarn generate:css -- --watch",
-    "dev:remix": "remix dev",
+    "dev:remix": "remix dev --port 3002",
```

do we need to specify port here? seems better to leave it how it is

```suggestion
    "dev:remix": "remix dev",
```